app/src/features/projects/view/ViewProjectPage.tsx

```diff
@@ -187,7 +187,8 @@ const ViewProjectPage: React.FC = () => {
   const completion_status =
     (end_date && moment(end_date).endOf('day').isBefore(moment()) && ProjectStatusType.COMPLETED) ||
     ProjectStatusType.ACTIVE;
-  const priority_status = ProjectStatusType.NOT_A_PRIORITY;
+  const priority_status =
+    projectWithDetails.location.priority === 'true' ? ProjectStatusType.PRIORITY : ProjectStatusType.NOT_A_PRIORITY;
```

Comment:
nit: could you move the enum values for priority into its own enum? The `ProjectStatusType` enum is really just about the project dates (is it still active, or not, or not even started (draft)).

Maybe define a new enum called `ProjectPriorityType` or similar

Reply:
Oh I see, its being used in the chip function below

Reply:
Okay, maybe lets just leave it as it is for now then.

Reply:
Hmm, we might have to look at how we are rendering the chips in general.  RIght now, if the project is not a priority, we still render a Box with mr={0.5}, even though its empty.